src/main/java/pl/zz/sharefood/controllers/FoodController.java

```diff
@@ -5,27 +5,39 @@
 import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.*;
 import pl.zz.sharefood.domain.Food;
+import pl.zz.sharefood.domain.dto.FoodDto;
+import pl.zz.sharefood.payload.PagedResponse;
 import pl.zz.sharefood.repository.FoodRepository;
+import pl.zz.sharefood.service.FoodGetService;
 import pl.zz.sharefood.service.FoodService;
 import pl.zz.sharefood.service.FoodDeleteService;
+import pl.zz.sharefood.util.AppConstants;
 
 import java.util.List;
 
 @RestController
-@RequiredArgsConstructor
```

Comment:
Dlaczego usunąłeś adnotację?

Reply:
a do czego ona jest wykorzystywana ?

Reply:
To jest adnotacja od lomboka i dzięki niej nie musisz tworzyć konstruktora